pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -179,3 +182,22 @@ func (o PauseConfig) RunPause() error {
 
 	return utilerrors.NewAggregate(allErrs)
 }
+
+// ObjectPauser is a function that marks the object in a given info as paused.
+// Returns the patched object in bytes and any error that occurred during the encoding or
+// in case the object is already paused.
+type ObjectPauser func(runtime.Object) ([]byte, error)
```

Comment:
polymorphic helpers need to be in the polymorphichelpers package so we can localize the debt that we have to pay down.  Otherwise we'll never find these again.  The package is a collection point for "things that should be generic but have ended up compiled in snowflakes"